metadata/site_import/sites/RefArch/payment-methods.xml

```diff
@@ -12,4 +12,10 @@
         <enabled-flag>true</enabled-flag>
         <processor-id>Adyen_Component</processor-id>
```

Comment:
in the description of adyen_component let's add a more clear description to differentiate it from credit cards. the description should say that this one is used for all alternative payment methods.

Reply:
Added descriptions.